test/wgen-codesonar-template.conf

```diff
@@ -3564,7 +3564,7 @@ SOURCE_MAX_REACHABLE_FROM_LIBRARIES = 100
 #  ABS_LOC for each and then printing its name.
 #
 #  The $str$Class_Name form used in some of the BAD_FUNCTION_MESSAGE
-#  factory settings is for internal GrammaTech use. Do not imitate
+#  factory settings is for internal CodeSecure use. Do not imitate
```

Comment:
Note that line 4283 of this conf file contains a Java import statement from some example code:

```java
    import com.grammatech.cs.*;
    import java.lang.*;
    ...
```

I left that line alone since it is still accurate in old versions of CodeSonar, this file is an old test file that is not in the compiled plugin, and there is currently no release version of CodeSonar where this namespace would exist.

Reply:
Makes sense to me